components/molecules/UniversalBenefitCard.js

```diff
@@ -39,9 +39,19 @@ export default function UniversalBenefitCard(props) {
     <div className={`benefit-card`} id={benefitCardId}>
       {props.benefit.statusCode != StatusCodes.activeAgreement ? (
         <StatusBadge
-          status={props.benefit.statusCode}
-          programCode={props.benefit.programCode}
-          locale={props.locale}
+          status={t[props.benefit.statusCode]}
```

Comment:
Don't forget to keep the sr-only text here
```suggestion
          status="<span className='sr-only'>{t[props.benefit.programCode]} </span>{t[props.benefit.statusCode]}"
```